core/controllers/feedback.py

```diff
@@ -33,13 +33,20 @@ class ThreadListHandler(base.BaseHandler):
 
     @acl_decorators.can_play_exploration
     def get(self, exploration_id):
+        suggestion_thread_dicts = (
+            [t.to_dict() for t in feedback_services.get_all_threads(
+                feconf.ENTITY_TYPE_EXPLORATION, exploration_id, True)])
+        suggestions = suggestion_services.get_suggestions_by_ids(
+            d['thread_id'] for d in suggestion_thread_dicts)
+        for thread_dict, suggestion in zip(
+                suggestion_thread_dicts, suggestions):
+            thread_dict['suggestion_dict'] = (
```

Comment:
No, don't do this please. Instead:

1) Write a PR that does two things: (a) populates SuggestionThreadModels with the relevant data, (b) includes a one-off job to run that adds this data to all existing models.

2) Get the one-off job run in production.

3) Write a separate PR which uses the new data (without doing any sort of manipulation in the controller layer).

Don't combine (1) and (3), we need (2) to run successfully first.